### DISCUSSION TITLE
EDGE is unsupported environment?

### DISCUSSION THREAD

**Discussion by zkailong** [coddingtonbear/obsidian-web#179] [Bug Report]
### Describe the bug

![image](https://github.com/coddingtonbear/obsidian-web/assets/13463791/ac8b291e-d122-45f1-98d7-903d3b699e52)

You are running Obsidian Web in an unsupported environment! Obsidian Web is tested only on recent versions of Chrome, and no guarantees are made by the maintainers of this project that it might work on any other browser â€” even other Chrome derivatives. Some features may not work correctly as a result, but you can feel free to [start a new discussion](https://github.com/coddingtonbear/obsidian-web/discussions/new?category=exotic-browser-support) if you run into difficulty â€” members of the community may be able to help!

![image](https://github.com/coddingtonbear/obsidian-web/assets/13463791/fe0a7a43-5181-4d27-8065-c72fb6d7a91d)

### To Reproduce

2. Click on '...'
3. Scroll down to '...'

### Operating System

windows 11

### Browser

Edge

### Browser Version

122.0.2365.80

**Comment by coddingtonbear** [comment]
Yes -- this message was written by me and is correct.  Edge is not a browser I will personally take the time to test this extension in, so you may experience bugs and are reliant on other members of the community for help if you run into trouble.  If the extension works there: great.  If it doesn't: sorry -- I do not have infinite time and will only personally offer help to users using this on a supported browser (just Chrome).  

I'm not sure how to make the message any clearer than that, but if you had a phrasing suggestion, feel free to post a pull request.
